pkg/controller/controller_instance_test.go

```diff
@@ -558,6 +559,80 @@ func TestReconcileInstanceAsynchronousNoOperation(t *testing.T) {
 	assertInstanceLastOperation(t, updatedInstance, "")
 }
 
+// TestReconcileInstanceAsynchronousUnsupportedBrokerError tests to ensure that, on an asynchronous
+// provision, an Instance's conditions get set with a Broker failure that is not one of the
+// "expected" response codes in the OSB API spec for provision.
+// See https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#response-2 for
+// the list of expected codes and a description of what we should do if another code is returned
+func TestReconcileInstanceAsynchronousUnsupportedBrokerError(t *testing.T) {
+	const (
+		brokerUsername = "testbrokeruser"
+		brokerPassword = "testbrokerpass"
+	)
+	controllerItems, err := newTestControllerWithBrokerServer(brokerUsername, brokerPassword)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer controllerItems.Close()
+
+	fakeKubeClient := controllerItems.FakeKubeClient
+	fakeCatalogClient := controllerItems.FakeCatalogClient
+	fakeBrokerServerHandler := controllerItems.BrokerServerHandler
+	testController := controllerItems.Controller
+	sharedInformers := controllerItems.Informers
+
+	fakeBrokerServerHandler.Catalog = fakebrokerserver.ConvertCatalog(getTestCatalog())
+
+	fakeKubeClient.AddReactor("get", "namespaces", func(action clientgotesting.Action) (bool, runtime.Object, error) {
+		return true, &v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				UID: types.UID("test_uid_foo"),
+			},
+		}, nil
+	})
+
+	sharedInformers.Brokers().Informer().GetStore().Add(getTestBroker())
+	sharedInformers.ServiceClasses().Informer().GetStore().Add(getTestServiceClass())
+
+	// Specify an error from the instance client
```

Comment:
is this the un'expected' error? can you add that word to match the comment above?